app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> We do still need a way to get a production-scaled (and in the case of a weekly calculation, "cut") curve for the must-run sources. I can look into that.

Cool, than I'll take a look at the influence the heat sources have to the business case calculation.